Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -7,16 +7,37 @@ namespace BootCamp.Chapter
     /// <summary>
     /// Caesar cipher which supports all ASCII character (256 in total)
     /// </summary>
-    public static class CaesarCipher
+    public class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            return EncryptString(message, shift);
         }
 
+        private static string EncryptString(string message, int shift)
+        {
+            if (message == null)
+            {
+                return null;
+            }
+
+            List<char> encryptString = new List<char>();
+
+            for (int j = 0; j < message.Length; j++)
+            {
+                var messageToInt = (int)message[j];
+                var messageToIntAddShift = (messageToInt + shift);
+                var messageEncrypt = (char)messageToIntAddShift;
+                encryptString.Add(messageEncrypt);
+                Console.WriteLine(messageEncrypt);
```

Comment:
This is not good, because there was no requirement for it. Maybe it was for you for debugging, but in actual code it shouldn't be there.

Reply:
It shouldn't be there, because it's a side effect of a function and this function now does 2 things.